resources/fetchers/process_fetcher.go

```diff
@@ -40,15 +43,65 @@ const (
 	CMDArgumentMatcher  = "\\b%s[\\s=]\\/?(\\S+)"
 	ProcessResourceType = "process"
 	ProcessSubType      = "process"
+	userHz              = 100
```

Comment:
The value of HZ varies across kernel versions and hardware platforms.
There is an assumption here for certain conditions that might not be respected between different deployments and versions.
I think that we can use https://man7.org/linux/man-pages/man3/sysconf.3.html and pull the value of `_SC_CLK_TCK` in order to reduce this assumption.

Reply:
Thank you for raising this up @jeniawhite .
It is a borrowed implementation from [go-sysinfo](https://github.com/elastic/go-sysinfo), we decided to deliver it with this known limitation while opening 2 new tickets to the go-sysinfo repo.
- https://github.com/elastic/go-sysinfo/issues/122
- https://github.com/elastic/go-sysinfo/issues/12

Once those tasks will be delivered, possibly by us, we'll change our implementation to use the go-sysinfo pkg.